parsec/core/mountpoint/fuse_runner.py

```diff
@@ -127,6 +130,21 @@ async def fuse_mountpoint_runner(
             encoding = sys.getfilesystemencoding()
 
             def _run_fuse_thread():
+                fuse_platform_options = {}
+                if sys.platform == "darwin":
+                    fuse_platform_options = {
+                        "local": True,
+                        "volname": workspace_fs.get_workspace_name(),
+                        "volicon": Path(resources.__file__).absolute().parent / "parsec.icns",
+                    }
+                    # osxfuse-specific options :
+                    # - local : allows mountpoint to show up correctly in finder (+ desktop)
+                    # - volname : specify volume name (default is OSXFUSE [...])
+                    # - volicon : specify volume icon (default is macOS drive icon)
+
+                elif sys.platform.startswith("linux"):
```

Comment:
no need to do startswith since python 3.3 (see https://docs.python.org/3/library/sys.html?highlight=sys#sys.platform), we can do `sys.platform == "linux"` ;-)

Reply:
...and we can just use `else` in fact here